packages/nepal-scan-scheduler/src/types/scanScopeItemAsset.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * ScanScheduler Service
+ * Advanced Scan Scheduling API
+ *
+ * OpenAPI spec version: 2.0
+ * Contact: support@alertlogic.com
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+import { ScanScopeItem } from './scanScopeItem';
+
+export interface ScanScopeItemAsset extends ScanScopeItem {
+    key: string;
+    type: ScanScopeItem.TypeEnum;
```

Comment:
Similar issue has been addressed in the following commit: https://algithub.pd.alertlogic.net/defender/scan_scheduler/commit/550f9b648e439e2e1f619a299c2abe6bfb70d3a7#diff-e487e0021bb79573e1160ab9db2ebc76R478

Let's see if by re-generating those files against the latest spec `type` will be correctly initialised.